packages/test/mocha-test-setup/mocharc-common.cjs

```diff
@@ -76,7 +76,7 @@ function getFluidTestMochaConfig(packageDir, additionalRequiredModules, testRepo
 		`configFile=${path.join(
 			__dirname,
 			"test-config.json",
-		)},cmrOutput=xunit+output+${outputFilePrefix}:mocha-json-output-reporter+output+${outputFilePrefix}:xunit+suiteName+${suiteName}`,
+		)},cmrOutput=xunit+suiteName+${suiteName}`,
```

Comment:
I don't think `cmrOutput` ends up being used. I believe it's an artifact of using [cypress-multi-reporters](https://www.npmjs.com/package/cypress-multi-reporters#cmroutput-option) at some point. I only find reference to `cypress` in property DDS, and it looks incomplete (it's in a mocha.opts file but there's no dependency).

That said, the update here looks wrong. I think we should either remove it fully or remove only the mocha-json-output-reporter options to have

```ts
`cmrOutput=xunit+output+${outputFilePrefix}:xunit+suiteName+${suiteName}`
```

Reply:
`xunit`'s output and suite name look reasonably set in `test-config.json`. I'd vote just remove this and confirm reports still look good in CI

Reply:
oh nevermind, `mocha-multi-reporters` uses the same option name (despite the acronym not being the same), see https://www.npmjs.com/package/mocha-multi-reporters#cmroutput-option. So probably needs an update

Reply:
ohhh yeah I remember how this works now. doc above explains it. The suite name gets substituted in as the value of `{id}` in `test-config.json`. So we do need both, and snippet above specifying the two bits for `xunit` is still required.